CSS/utility.css

```diff
@@ -1,27 +1,27 @@
 @import url('https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@200;300;400;600&family=Roboto:wght@300&display=swap');
 
 .py-1x {
-  padding: 10px;
+  padding: 0.625rem;
 }
 
 .py-2x {
-  padding: 20px;
+  padding: 1.25rem;
 }
 
 .py-3x {
-  padding: 30px;
+  padding: 1.875rem;
 }
 
 .mg-1x {
-  margin: 10px;
+  margin: 0.625rem;
 }
```

Comment:
Here you can write padding-1 or p-1 and margin-1 or m-1 as py usually means vertical padding and you can remove x from padding and margin (py-1x/2x/3x and just keep p-1/2/3)

Reply:
Rest looks good to me

Reply:
I got your point but I have used py-1x/2x/3x in all my components, I will update the recommended changes once I finish all my components so that I don't get merge conflicts.